content/blogs/newsletter-104.md

```diff
@@ -0,0 +1,90 @@
+
+With that said, the ReactOS Team is pleased to announce the upcoming of 4 new developers for the project -- Carl J. Bialorucki (cbialorucki), Doug Lyons (Doug-Lyons), Oleg Dubinskiy (oleg-dubinskiy) and Whindmar Saksit (whindsaks).
+Their main focus is to maintain the Shell and Audio components respectively, as well as on any module that requires assistance.
+
+## Symmetric Multiprocessing (SMP) support status & GCC 13 preparations
+The symmetric multiprocessing support for AMD64 architecture is getting extensively improved thanks to Timo Kreuzer (tkreuzer).
+In particular, he has implemented code for processor freezing and switching ([PR #6581](https://github.com/reactos/reactos/pull/6581)) so that every logical processor is frozen or switched properly.
+In addition he implemented basic support for IPIs (interprocessor interrupts) which are needed for SMP scheduling, debugging and whatnot ([PR #6130](https://github.com/reactos/reactos/pull/6130)).
+These are only limited to APC, DPC and freeze IPIs for the moment being.
+
+Another important improvement in the area of SMP is the kernel initialization in the AMD64 build architecture, which got refurbished by Timo ([PR #6110](https://github.com/reactos/reactos/pull/6110)).
+This allows the SMP support to work properly.
+
+Timo also helped fixing some bugs in order to get ReactOS compile-able for GCC 13 build environment.
+Most of the chunk of GCC 13 preparations goes thanks to Simone Mario Lombardo (simonelombardo) for making ReactOS bootable.
+
+In the meantime Justin Miller (The_DarkFire) is focusing on the x86 architecture of the SMP area.
+He worked on the preparations for the application processor bootup ([PR #5879](https://github.com/reactos/reactos/pull/5879)) in x86 SMP HAL and is planning to further improve the x86 part of SMP.
+
+**It is worth noting that SMP is still heavily under development and there is no exact estimation when will the multicore support finally come out as ready, so stay tuned for more updates!**
+
+## User-mode WINE modules syncing
+Timo, Justin and other contributors are making an effort to bring up up-to-date some of the user-mode modules that we borrow from WINE.
+Specifically the components that we do share with WINE are KERNEL32, COMCTL32, USER32 and others of which were left in an obsolete state.
+
+The aim of this effort is to bring the improvements and fixes from WINE that help with the support of NT6+ software that otherwise wouldn't have worked.
+This also helps getting rid of hacks introduced.
+
+## Audio improvements
+Several major improvements in audio subsystem have been made by Oleg Dubinskiy recently.
+In particular, he implemented handling modern audio formats (e.g., Extensible audio, Float audio and others), [PR #6686](https://github.com/reactos/reactos/pull/6686), which support high sample rates, bits per sample sound quality and more than two (stereo) output channels.
```

Comment:
```suggestion
In particular, he implemented handling modern audio formats (e.g. Extensible audio, Float audio, and others), [PR #6686](https://github.com/reactos/reactos/pull/6686), which support high sample rates, bits per sample sound quality and more than two (stereo) output channels.
```

Reply:
Review comment has been marked as addressed, but it's not... there should not be a comma right after the "e.g."

Reply:
Then next time I advise everyone to not review the very same thing which only confuses and overwhelms me when addressing reviews. Literally there's like 3 reviews with mixed up suggestions for the exact same thing... that's not helpful.

Reply:
It's hard for reviewer to track review comments of other reviewers.